tests/integration/025_notifications/main.tf

```diff
@@ -11,9 +11,10 @@ locals {
 }
 
 resource "env0_notification" "test_notification_1" {
-  name  = "${local.notification_name_prefix}-1-${random_string.random.result}"
-  type  = "Slack"
-  value = "https://someurl1.com"
+  name           = "${local.notification_name_prefix}-1-${random_string.random.result}"
+  type           = "Slack"
```

Comment:
should be `type = "Webhook"`

Reply:
aligned with `type` body param [here](https://docs.env0.com/reference/notifications-create-notification-endpoint)